engine/src/test/java/io/zeebe/engine/processor/workflow/incident/ServiceTaskIncidentTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.processor.workflow.incident;
+
+import io.zeebe.engine.util.EngineRule;
+import io.zeebe.engine.util.client.DeploymentClient;
+import io.zeebe.model.bpmn.Bpmn;
+import io.zeebe.model.bpmn.BpmnModelInstance;
+import io.zeebe.model.bpmn.builder.ServiceTaskBuilder;
+import io.zeebe.protocol.record.Assertions;
+import io.zeebe.protocol.record.Record;
+import io.zeebe.protocol.record.RecordType;
+import io.zeebe.protocol.record.RejectionType;
+import io.zeebe.protocol.record.intent.DeploymentIntent;
+import io.zeebe.protocol.record.value.DeploymentRecordValue;
+import io.zeebe.test.util.record.RecordingExporter;
+import io.zeebe.test.util.record.RecordingExporterTestWatcher;
+import java.util.Optional;
+import java.util.function.Consumer;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class ServiceTaskIncidentTest {
+
+  @ClassRule public static final EngineRule ENGINE = EngineRule.singlePartition();
+
+  private static final String PROCESS_ID = "process";
+
+  @Rule
+  public final RecordingExporterTestWatcher recordingExporterTestWatcher =
+      new RecordingExporterTestWatcher();
+
+  private static BpmnModelInstance workflow(final Consumer<ServiceTaskBuilder> consumer) {
+    final var builder = Bpmn.createExecutableProcess(PROCESS_ID).startEvent().serviceTask("task");
+
+    consumer.accept(builder);
+
+    return builder.endEvent().done();
+  }
+
+  @Test
+  public void shouldRejectDeploymentfJobTypeExpressionCanNotBeEvaluated() {
```

Comment:
The other incident tests verify that an incident is created and can be resolved. Incidents are created at runtime when the workflow instance is executed. In case of the job type, an incident would be created if the expression fails (e.g. no variable found), or if it evaluates not to a string.

This test case verifies that the validation works. This is already tested in `ZeebeRuntimeValidationTest`.

Reply:
done, please check again